### PR TITLE
Fix column order and bounds.

### PR DESCRIPTION
**Pull request by oerling**
Partitioning columns are always supposed to be after all other
columns. Aria filter pushdown expects this to be the case and this is
also enforced at table creation time. Hence we make the test also
comply with this. Since Aria returns larger batches, the TableScan
test goes through its first 20K rows in one call. This means that the
system memory drops faster, hence limits are updated.

### CODE REVIEW

File: presto-hive/src/test/java/com/facebook/presto/hive/TestOrcPageSourceMemoryTracking.java [hunk 1]
```diff
@@ -338,10 +339,15 @@ public void testTableScanOperator()
             assertFalse(operator.isFinished());
             Page page = operator.getOutput();
             assertNotNull(page);
-            page.getBlock(1);
+            page.getBlock(0);
             if (memoryUsage == -1) {
                 memoryUsage = driverContext.getSystemMemoryUsage();
-                assertBetweenInclusive(memoryUsage, 460000L, 469999L);
+                if (SystemSessionProperties.isAriaScanEnabled(driverContext.getSession())) {
```
Comment (mbasmanova) [inline presto-hive/src/test/java/com/facebook/presto/hive/TestOrcPageSourceMemoryTracking.java]: static import

Comment (mbasmanova) [reply presto-hive/src/test/java/com/facebook/presto/hive/TestOrcPageSourceMemoryTracking.java]: @oerling Orri, could you make a change to statically import SystemSessionProperties.isAriaScanEnabled ?

**Comment by oerling** [comment]
OK

**Comment by mbasmanova** [comment]
@oerling there is a style check error:

```
[ERROR] src/test/java/com/facebook/presto/hive/TestOrcPageSourceMemoryTracking.java:[94] (imports) ImportOrder: Wrong order for 'com.facebook.presto.hive.HiveColumnHandle.ColumnType.PARTITION_KEY' import.
```

**Comment by oerling** [comment]
This produces an error 

 

[INFO] Running com.facebook.presto.hive.TestOrcPageSourceMemoryTracking

[ERROR] Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 4.183 s <<< FAILURE! - in com.facebook.presto.hive.TestOrcPageSourceMemoryTracking

[ERROR] testTableScanOperator(com.facebook.presto.hive.TestOrcPageSourceMemoryTracking)  Time elapsed: 0.045 s  <<< FAILURE!

com.facebook.presto.spi.PrestoException: Unknown session property: null.aria_scan_enabled

      at com.facebook.presto.FullConnectorSession.getProperty(FullConnectorSession.java:129)

      at com.facebook.presto.hive.HiveSessionProperties.isAriaScanEnabled(HiveSessionProperties.java:587)

      at com.facebook.presto.hive.TestOrcPageSourceMemoryTracking.testTableScanOperator(TestOrcPageSourceMemoryTracking.java:345)

      at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

      at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

 

 

What is the deal about the prefix of the property name?

**Comment by mbasmanova** [comment]
@oerling 

> What is the deal about the prefix of the property name?

The prefix is the catalog name. Looks like the session doesn't have the catalog, e.g. it is null.

**Comment by mbasmanova** [comment]
Closing stale PR on behalf of @oerling.

**Pull request closed without merging**